assets/stylesheets/wizard/custom/base.scss

```diff
@@ -80,3 +80,7 @@ textarea {
   }
 }
 
+.body-length {
+  text-align: right;
+  font-weight: bold;
```

Comment:
@fzngagan No need for this to be bold. Users can bold it in a theme if they wish.